app/questionnaire/placeholder_transforms.py

```diff
@@ -60,14 +60,12 @@ def format_date(self, date_to_format: str, date_format: str) -> str:
     @staticmethod
     def format_list(list_to_format: Sequence[str]) -> str:
         formatted_list = "<ul>"
-        for item in list_to_format:
-            formatted_list += f"<li>{item}</li>"
+        formatted_list += "".join([f"<li>{item}</li>" for item in list_to_format])
```

Comment:
```
        formatted_list = "<ul>" + "".join(
            [f"<li>{item}</li>" for item in list_to_format]
        )
```

Reply:
Updated the `format_list` to your suggested code👍